### ISSUE TITLE
nav bar alignment

### ISSUE THREAD

**Issue by Gamerking177** [ML-Fusion-Lab/ML-Fusion-Lab-Website#45]
![image](https://github.com/user-attachments/assets/f0866b40-36d3-4f01-ad05-015dbe9b6f1e)

**Comment by Iampratyush004** [comment]
please assign this to me , as i have a ready to push code with adjusted alignment

**Comment by vivekvardhan2810** [comment]
@Gamerking177 will it be okay for you if i assign it to @Iampratyush004

**Comment by Iampratyush004** [comment]
Kindly Assign me sir

**Comment by Iampratyush004** [comment]
Sir your project is not reflecting on hacktober

**Comment by vivekvardhan2810** [comment]
yes, i will remove the label @Iampratyush004

**Comment by Iampratyush004** [comment]
Sir the code is becoming a bit tedious as some css is writing in style.css and some in style tag under html .

We should be css in file only or their respective css file so there is a consistency and easy to make changes...as common elements style in main style.css file and page specific will be stored in page specific 
@vivekvardhan2810 
Please look into the matter

**Comment by vivekvardhan2810** [comment]
okay @Iampratyush004, what's your solution on it.

**Comment by Iampratyush004** [comment]
No please dont remove label...as i will align nav bar now ...but for long term...there could be an issue

**Comment by vivekvardhan2810** [comment]
but i didn't registered this project under hacktoberfest @Iampratyush004

**Comment by Iampratyush004** [comment]
ok sir

**Comment by Iampratyush004** [comment]
@vivekvardhan2810  sir i have a pull request regarding this issue. plz see to it

**Comment by vivekvardhan2810** [comment]
Okay @Iampratyush004 I Will look on it soon.

**Comment by Gamerking177** [comment]
> @Gamerking177 will it be okay for you if i assign it to @Iampratyush004

Sir it is unfair I will complain to gssoc at discord

**Comment by vivekvardhan2810** [comment]
Okay @Gamerking177 sorry.

**Comment by vivekvardhan2810** [comment]
@Iampratyush004 as you have seen the message from @Gamerking177, pls do the issue which i have assigned you before.

**Comment by aditya-bhaumik** [comment]
@Gamerking177 hi you can work on this issue 

You can let me know about your complains also join the discord server for communication

**Comment by Iampratyush004** [comment]
> @Iampratyush004 as you have seen the message from @Gamerking177, @Iampratyush004 pls do the issue which i have assigned you before.

sir this is unfair, and very unethical .. i have already pull request and done the changes required .. u had put me on hold and now that you have the request u are assigning to somebody else. I am sorry to say but u are questioning my   potential @vivekvardhan2810

**Comment by Iampratyush004** [comment]
@vivekvardhan2810  there is already a pull request pending on this issue.. instead of reviewing u had unassigned from this task

**Comment by Iampratyush004** [comment]
> @Iampratyush004 as you have seen the message from @Gamerking177, @Iampratyush004 pls do the issue which i have assigned you before.

and sir could i know which previous task u have metioned.. my issue number #66 is still pending to be assigned , u have said to do this first

**Comment by vivekvardhan2810** [comment]
@Iampratyush004 wait till tomorrow i will assign you, I will make sure you that.

**Comment by Iampratyush004** [comment]
please assign me this and pls tag hacktober fest , gssoc-extd and level 2

**Comment by vivekvardhan2810** [comment]
> please assign me this and pls tag hacktober fest , gssoc-extd and level 2

for which issue @Iampratyush004

**Comment by Iampratyush004** [comment]
this #45

**Comment by vivekvardhan2810** [comment]
> please assign me this and pls tag hacktober fest , gssoc-extd and level 2

@Iampratyush004 as i have said early this project is not under hacktoberfest and i can't give you that label.

**Comment by vivekvardhan2810** [comment]
@Iampratyush004 Can you communicate with me this number 9492962646, because my internet is slow.

**Comment by aditya-bhaumik** [comment]
@vivekvardhan2810 you can

**Comment by Iampratyush004** [comment]
sir hacktober fest is under reviewing the previous one and hopefully it will be under hacktober fest in 7 days

**Comment by aditya-bhaumik** [comment]
@Iampratyush004 as you have already created a pull request @vivekvardhan2810 please review his pull request also and make sure you dont make this mistake again

**Comment by Iampratyush004** [comment]
> @Iampratyush004 @Gamerking177 dont worry i told vivek to learn from his mistake he is also human he will not make this mistake again
> 
> Meanwhile you both can create pull request for this issue he will give labels to both of you
> 
> You both will get the points dont worry

ok sir thank you sir
and sir as this issue is raised to level2 , pls make sure the labels mentioned are gssoc-extd, hacktoberfest and level2
thank you sir

**Comment by aditya-bhaumik** [comment]
@vivekvardhan2810 make sure it is added in both of their pull requests

**Comment by vivekvardhan2810** [comment]
Okay @vivekvardhan2810

**Comment by Gamerking177** [comment]
Sorry

**Comment by vivekvardhan2810** [comment]
> @vivekvardhan2810 I will go to dc to complain about u

@Gamerking177 sorry it's mistake from my side, from next i won't repeat such mistakes.

**Comment by Gamerking177** [comment]
Sorry

**Comment by Gamerking177** [comment]
Sir

**Comment by vivekvardhan2810** [comment]
@Gamerking177 Sorry from my side pls don't complain, i won't repeat it again

**Comment by Gamerking177** [comment]
> > @vivekvardhan2810 I will go to dc to complain about u
> 
> @Gamerking177 sorry it's mistake from my side, from next i won't repeat such mistakes.

No sir I not seen u will assign me also

**Comment by Gamerking177** [comment]
Sorry sir sorry

**Comment by Gamerking177** [comment]
> @Gamerking177 Sorry from my side pls don't complain, i won't repeat it again

Sorry sir sorry

**Comment by Gamerking177** [comment]
@vivekvardhan2810 thanks sir for assign and sorry sir

**Comment by vivekvardhan2810** [comment]
> @vivekvardhan2810 thanks sir for assign and sorry sir

@Gamerking177 it's okay, No problem.

**Comment by Kavish-Paraswar** [comment]
"I would like to work on this issue as I have a strong background in CSS and HTML. Please assign me to this task under GSSOC'24 Extended."

**Comment by vivekvardhan2810** [comment]
@Kavish-Paraswar this issue was assigned before, pls don't spam.

**Comment by aditya-bhaumik** [comment]
@Kavish-Paraswar i noticed that you are spamming in all the issues of all the project i warned you already in my repo 

I already warned you multiple times not to spam 

If you dont stop even after multiple warning strict action will be taken

**Comment by Gamerking177** [comment]
@vivekvardhan2810 pls check this pr sir and merge it
